### PR TITLE
Correct minor cetc

### PR DESCRIPTION
**Pull request by vfarinhaluz**
Corrected typo and removed redundant information.

### CODE REVIEW

**Comment by vfarinhaluz** [comment]
Sorry for the multiple pull requests. We should be all set now!
Thank you for setting this up. It’s become my go-to place for seminar information.
Vitor

File: conferences.md [hunk 1]
```diff
@@ -12,21 +12,19 @@ This year's conference is organized by the University of British Columbia.
 
 **Time:** 7am to 5:20pm PST
 
-**Program:** [available here.](https://microeconomics.ca/micro/cetc)
+**Program and instructions:** [available here.](https://microeconomics.ca/micro/cetc)
```
Comment (just-cameron) [inline conferences.md]: This should be capitalized:

```suggestion
**Program and instructions:** [Available here](https://microeconomics.ca/micro/cetc).
```

File: conferences.md [hunk 2]
```diff
@@ -12,21 +12,19 @@ This year's conference is organized by the University of British Columbia.
...
 
-**Access:** Open to the public. [Please register here.](https://zoom.us/webinar/register/9615872292160/WN_ZtYIUomiRPigDbo28x1sdg)
+**Access:** open to the public. [Please register here.](https://zoom.us/webinar/register/9615872292160/WN_ZtYIUomiRPigDbo28x1sdg)
```
Comment (just-cameron) [inline conferences.md]: This should also stay capitalized. It's also much more clear if you separate the registration link from "Access":

```suggestion
**Access:** Open to the public. 

**Registration**: [Please register here](https://zoom.us/webinar/register/9615872292160/WN_ZtYIUomiRPigDbo28x1sdg).
```

**Review by just-cameron — COMMENTED** [review]
I left two comments. I'll merge if you don't want to take the suggestions, though, as it's up to you.

**Comment by vfarinhaluz** [comment]
The changes are great. Thank you!

**Pull request merged**